astropy/coordinates/tests/test_angles.py

```diff
@@ -3,8 +3,6 @@
 
 # TEST_UNICODE_LITERALS
```

Comment:
Shouldn't we remove that too? Or should that be done in a seperate PR?

Reply:
There still need to be a big sweep for testing unicode literals stuff because it is used in several subpackages (and also the test helper).

Reply:
The "TEST_UNICODE_LITERALS" strings can indeed be removed, it's useful on Python 3.

Reply:
If they are useful why remove them? 😄

Reply:
Oops :rofl: